devicemodel/hw/pci/wdt_i6300esb.c

```diff
@@ -41,15 +41,15 @@
 #define ESB_WDT_REBOOT	(0x01 << 5)   /* Enable reboot on timeout */
 #define ESB_WDT_RELOAD	(0x01 << 8)   /* Ping/kick dog  */
 #define	ESB_WDT_TIMEOUT	(0x01 << 9)   /* WDT timeout happened? */
-#define TIMER_TO_SECONDS(val)	(val >> 9)
+#define TIMER_TO_SECONDS(val)	(val >> 10) /* 10bits for 1s in a 1KHz Clock */
 
 /* Magic constants */
 #define ESB_UNLOCK1	0x80   /* Step 1 to unlock reset registers  */
 #define ESB_UNLOCK2	0x86   /* Step 2 to unlock reset registers  */
 
 #define WDT_TIMER_SIG			0x55AA
```

Comment:
SIGNATURE?  use full name?

Reply:
This is Minggui's original work and would be removed if my WDT INTR enabling patch get merged, so I want to keep it for now.